pubspec.yaml

```diff
@@ -4,7 +4,7 @@ author: Dart Team <misc@dartlang.org>
 description: Automatic sourcecode generation for Dart
 homepage: https://github.com/dart-lang/source_gen
 environment:
-  sdk: '>=1.12.0 <2.0.0'
+  sdk: '>=1.23.0-dev.11.0 <2.0.0'
```

Comment:
Probably need both? SDK for supporting the output in tools/runtimes,  analyzer for supporting parsing this file in later codegen steps etc?